app/Actions/Database/StopDatabaseProxy.php

```
@@ -22,16 +23,69 @@ class StopDatabaseProxy
+        $this->dispatchDatabaseProxyStoppedEvent();
+
+    }
+
+    protected function runRemoteCommands(array $commands, Server $server, bool $throwError = true): ?string
+    {
+        return instant_remote_process($commands, $server, $throwError);
+    }
+
+    protected function dispatchDatabaseProxyStoppedEvent(): void
+    {
         DatabaseProxyStopped::dispatch();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Events\DatabaseProxyStopped' in method 'dispatchDatabaseProxyStoppedEvent'. (undefined)

(StaticAccess)